src/open_samus_returns_rando/files/templates/randomizerpowerup.lua

```diff
@@ -217,6 +194,40 @@ function RandomizerPowerup.IncreaseAmmo(resource)
     RandomizerPowerup.IncreaseItemAmount(current_id, resource.quantity, resource.item_id)
 end
 
+function RandomizerPowerup.ObjectiveComplete()
+    if RandomizerPowerup.GetItemAmount("ITEM_ADN") == (39 + Init.iExtraDNA) then
```

Comment:
Oh. When I was reading the schema, description and names etc. I really thought that "extra" means placed DNA but not required (and was wondering why we would even need that value). Reading this line means, extra DNA must be collected.
Maybe it's just me connecting "extra" always more towards the meaning of something like "bonus".

Dunno how the RDV PR will look like but extra DNA really only makes sense when you want to collect more than 39 DNA.
Which makes me wonder: Do we really still want to count up if the value is not fixed anymore to 39? Because we never show how many DNA you need to collect due to not enough space in the lower bar (which was fine when it always needs to be 39).

Reply:
Honestly, the use case of this is so niche that I might just remove it. If you wanted more dna placed you don't really need this plus this could be checked for a different way.

Reply:
> Dunno how the RDV PR will look like but extra DNA really only makes sense when you want to collect more than 39 DNA. Which makes me wonder: Do we really still want to count up if the value is not fixed anymore to 39? Because we never show how many DNA you need to collect due to not enough space in the lower bar (which was fine when it always needs to be 39).

I would prefer to just have the dna count start at what you need to finish (the required dna) and then count down to 0, but what I tried didn't work and I didn't look too hard, so the behavior is the same as before. Maybe it's trivial to change it and I'm not realizing it.